selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java

```diff
@@ -139,11 +111,11 @@ public void shouldDisplayJavaDocOfClassMethod() throws Exception {
         "concat(String part1, String part2, char divider) : String App");
 
     // then
-    checkProposalDocumentationHTML(expectedJavadocHtmlText);
+    checkProposalDocumentationHTML(getJavadocHtmlText());
```

Comment:
`getExpectedJavadocHtmlText()` method name more relevant to method intention, IMHO.